### ISSUE TITLE
Create Machine Learning Tutorials with Flux.jl

### ISSUE THREAD

**Issue by logankilpatrick** [FluxML/fluxml.github.io#107]
Hello prospective hacktoberfest contributor! The FluxML community would welcome new tutorials to the Flux website which can generally be found under: https://fluxml.ai/tutorials.html

You can find the source code for the tutorials here: https://github.com/FluxML/fluxml.github.io/tree/main/tutorials/_posts. They are just markdown files. 

## What we are looking for

We would be open to Pull Request which provide a tutorial topic that is not already covered but the existing tutorials. But no need to re-invent the wheel here. If you have a favorite tutorial that you want to try and re-create using Flux, we would love to help and see it! 

Find out more about contributing here: https://github.com/FluxML/fluxml.github.io/blob/main/CONTRIBUTING.md and more general ways of contributing (which may not be open hacktoberfest issues but we can happily make them into issues if that helps you) here: https://github.com/FluxML/Flux.jl/blob/master/CONTRIBUTING.md

**Comment by Dsantra92** [comment]
I would love to see some of my favourite tutorials in Tensorflow and PyTorch for Flux. I propose to write flux versions for these tutorials:
- [A simple GAN for generation of MNIST Images](https://www.tensorflow.org/tutorials/generative/dcgan)
- [Neural transfer](https://pytorch.org/tutorials/advanced/neural_style_tutorial.html)
- [Simple object detection](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html)

I would love to hear your feedback on these tutorials.

P.s: These are tutorials than I can think off the top of my head. If these PRs go well I would love to add more tutorials in the future.

**Comment by logankilpatrick** [comment]
> I would love to see some of my favourite tutorials in Tensorflow and PyTorch for Flux. I propose to write flux versions for these tutorials:
> 
> * [A simple GAN for generation of MNIST Images](https://www.tensorflow.org/tutorials/generative/dcgan)
> * [Neural transfer](https://pytorch.org/tutorials/advanced/neural_style_tutorial.html)
> * [Simple object detection](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html)
> 
> I would love to hear your feedback on these tutorials.
> 
> P.s: These are the tutorials that I can think off the top of my head. If these PRs go well I would love to add more tutorials in the future.

This would be incredible! Let us know how we can help.

**Comment by kailukowiak** [comment]
@logankilpatrick I've been working on converting [this tutorial](https://pytorch.org/tutorials/intermediate/char_rnn_classification_tutorial.html) tutorial to Julia. I've got a working version for the CPU but my `rnn` uses a loop with indexes.

```
function rnn(input, hidden, model)
    combined = [input; hidden] |> model.device
    out = model.in2out(combined)
    hidden = model.in2hidden(combined)
    return out, hidden
end
```
This throws an error when I try and take the gradient and straight-up failes if I set: `CUDA.allowscalar(false)`. I could use the build in flux functions but I wanted to stay closer to the method in the tutorial as I think it gives good intuition into what's actually going on. Do you have any idea how I could get around the issues with integer slicing a GPU array?

**Comment by ToucheSir** [comment]
Hi @kailukowiak, this kind of question is better suited for Discourse (we try to keep the issue tracker to bugs and feature requests). If you wouldn't mind opening a thread there (there's a Github login option) and letting us know, we can pick up there.
